Sources/NIOSSL/CustomPrivateKey.swift

```diff
@@ -75,12 +80,14 @@ public protocol NIOSSLCustomPrivateKey: _NIOPreconcurrencySendable {
     func decrypt(channel: Channel, data: ByteBuffer) -> EventLoopFuture<ByteBuffer>
 }
 
+extension NIOSSLCustomPrivateKey {
+    var derBytes: [UInt8] { [] }
```

Comment:
I'm pretty sure this has to be public, and it escaped only because we have a `@testable` import in the test file.